inc/class-ubc-isp-sub-menu-walker.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Register a custom navigation walker class
+ *
+ * @package UBC Collab ISP Child Theme Functions
+ */
+class UBC_ISP_Sub_Menu_Walker extends Walker_Nav_Menu {
+
+	/**
+	 * Summary of start_el
```

Comment:
I'm not normally TOO pushy on this, but when it comes to modifying menus etc. please could you write doc blocks for these functions which which outline what they do/what you're changing etc? Future me thanks you in advance that I don't have to work out what is going on here.

Reply:
Thanks for this. Using a full Walker for what I wanted was actually more than I needed. Instead I've added some filters into the page-header.php function class that should add a toggle button for menu items with child links and add description text above sub menu items with descriptions.